config/config.go

```diff
@@ -439,12 +439,14 @@ func (s *Security) ClusterSecurity() tikvcfg.Security {
 
 // Status is the status section of the config.
 type Status struct {
-	StatusHost      string `toml:"status-host" json:"status-host"`
-	MetricsAddr     string `toml:"metrics-addr" json:"metrics-addr"`
-	StatusPort      uint   `toml:"status-port" json:"status-port"`
-	MetricsInterval uint   `toml:"metrics-interval" json:"metrics-interval"`
-	ReportStatus    bool   `toml:"report-status" json:"report-status"`
-	RecordQPSbyDB   bool   `toml:"record-db-qps" json:"record-db-qps"`
+	StatusHost           string `toml:"status-host" json:"status-host"`
+	MetricsAddr          string `toml:"metrics-addr" json:"metrics-addr"`
+	StatusPort           uint   `toml:"status-port" json:"status-port"`
+	MetricsInterval      uint   `toml:"metrics-interval" json:"metrics-interval"`
+	ReportStatus         bool   `toml:"report-status" json:"report-status"`
+	RecordQPSbyDB        bool   `toml:"record-db-qps" json:"record-db-qps"`
+	GrpcKeepAliveTime    uint   `toml:"grpc-keepalive-time" json:"grpc-keepalive-time"`
+	GrpcKeepAliveTimeout uint   `toml:"grpc-keepalive-timeout" json:"grpc-keepalive-timeout"`
```

Comment:
Can we have a test to verify this behavior? (You can simulate a TCP packet dropping by sending via your MITM proxy)

Reply:
Yes, I have done it. It behaved as expected.